### ISSUE TITLE
Possibility to include a English txt

### ISSUE THREAD

**Issue by MrCampy** [BeniYukiMai/MaiDumpTool#12]
Could you include the possibility for translate the app only including a English txt, please?

**Comment by realshotgun** [comment]
It gets an english update within an hour of Mai releasing the tool.  Not sure why he would need to do it himself.

**Comment by Dasanko** [comment]
Should just read the strings from a plain text UTF-8 or UTF-16 file, something like:
cn_zh.txt or en_us.txt

**Comment by MrCampy** [comment]
Yes... somebody can translate to Spain, Italian, French, etc

**Comment by javo92** [comment]
True, besides, if the app gets updated it could keep backward compatibility of the different translations

**Comment by panther666** [comment]
+1 for english language from author

**Comment by holyst4r** [comment]
Mai could just collaborate with a user to provide an english translation for the interface, changelog can be translated later ...

**Comment by chrisfu** [comment]
+1 for this. Shouldn't be too difficult to include translation files at `lang/en.json` or `lang/fr.json` or something like that. I'm sure there'll be plenty of folk willing to provide translations.

**Comment by Keizel** [comment]
Please, @BeniYukiMai , you must think it!
